doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -245,41 +241,12 @@ Bluetooth Fast Pair samples
 Bluetooth Mesh samples
 ----------------------
 
-* Added:
```

Comment:
Entries of Bluetooth Fast Pair samples could be removed (line no:237-239).

Reply:
Seems like this is not yet fixed.

Reply:
it is fixed now